packages/web/public/static/locales/zh_CN/common.json

```diff
@@ -121,7 +121,8 @@
       "threadCommentThanks": "{{userIdentifier}} 感谢了您的点评",
       "threadCommentThanks_plural": "{{userIdentifier}} 感谢了您的 {{count}} 条点评",
       "newPosts": "您关注的 1 位Journaler发布了新日志",
-      "newPosts_plural": "您关注的Journalers发布了 {{count}} 篇新日志"
+      "newPosts_plural": "您关注的Journalers发布了 {{count}} 篇新日志",
+      "mention": "{{mentioner}} mentioned you in a comment"
```

Comment:
@pushinhuang just letting you know this new translation string is here in case you have a few minutes to help us translate it! 🙏🏼😄 No worries at all if not - we're planning on wrapping this up in 1 week 🚀